activerecord/lib/active_record/enum.rb

```diff
@@ -75,6 +75,19 @@ module ActiveRecord
   #
   #   Conversation.where("status <> ?", Conversation.statuses[:archived])
   #
+  # You can use <tt>:prefix</tt> option then you need to define multiple enums
+  # with same values. If prefix is <tt>true</tt>, the methods are prefixed with
+  # the name of the enum.
+  #
+  #   class Invoice < ActiveRecord::Base
+  #     enum({verification: [:done, :fail]}, prefix: true)
```

Comment:
This could easily be written (and be more readable) as:

```ruby
class Invoice < ActiveRecord::Base
  enum status: [:unverified, :verified]
end
```

Can we find a better use case for prefix than the above?